controllers/cloud.redhat.com/clowdapp_controller.go

```diff
@@ -140,6 +140,11 @@ func (r *ClowdAppReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{Requeue: true}, err
 	}
 
+	if env.Generation != env.Status.Generation {
+		r.Recorder.Eventf(&app, "Warning", "ClowdEnvNotReconciled", "Clowder Environment [%s] has not been reconciled", app.Spec.EnvName)
```

Comment:
I kind of think we shouldn't record an event if this happens.  We're just properly preventing a race condition here, right?